charts/kasm-ws/README.md

```
@@ -0,0 +1,2596 @@
+
+## Value type details
+
+The chart uses some value types for which I would like to add a further explanation / recommendation:
+
+### string/runtimeClassName
+
+Provide a valid name of a runtimeClass within your cluster.
+
+If you want to get the current available runtimeClasses within your cluster, exec:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)